addons/hr/static/src/mixins/chat_mixin.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+const { Component } = owl;
+
+export const EmployeeChatMixin = {
+    async openChat(ev) {
+        ev.preventDefault();
+        ev.stopPropagation();
+
+        const messaging = await Component.env.services.messaging.get();
+        messaging.openChat({ employeeId: this.getEmployeeId(ev) });
+    },
```

Comment:
I would remove the event from the argument and just pass the employeeId
```suggestion
    async openChat(employeeId) {
        const messaging = await Component.env.services.messaging.get();
        messaging.openChat({ employeeId });
    },
```